bin/oref0-setup.sh

```diff
@@ -745,8 +746,9 @@ if prompt_yn "" N; then
 
     echo Checking oref0 installation
     cd $HOME/src/oref0
+    nvm use 15.14.0
```

Comment:
Why wouldn't we just execute this immediately after installing that version?

Reply:
This was just me testing that the node version was actually correctly updated, the install should have been working in each shell but for some reason kept coming against npm stating older versions of node being used (which as you've said is an issue with nvm not working for root)